tests/aws/services/firehose/test_firehose.py

```diff
@@ -532,3 +532,44 @@ def test_kinesis_firehose_kinesis_as_source_multiple_delivery_streams(
             ]
         )
         snapshot.match("kinesis-event-stream-multiple-delivery-streams", s3_data)
+
+    @markers.aws.unknown
```

Comment:
we're not accepting any more `unknown` tests. Please use the `TEST_TARGET` env variable to run agains the AWS. This will generate and extra file that you need to also submit in this pr. Also change it to `@markers.aws.validated` once you confirm that the test passes with AWS . Please check these [docs](https://docs.localstack.cloud/contributing/) for more info.